src/renderer/widgets/Staking/Nominate/default/model/form-model.ts

```diff
@@ -0,0 +1,356 @@
+  transaction: $coreTx,
+});
+
+const $isMultisig = $route.map((route) => {
+  return route.some((acc) => accountUtils.isMultisigAccount(acc));
+});
+
+const $proxyAccount = combine({ route: $route }, ({ route }) => {
+  return route.find(accountUtils.isProxiedAccount) ?? null;
+});
+
+const $isProxy = $proxyAccount.map((account) => {
+  return nonNullable(account);
+});
+
+const $proxyWallet = combine(
+  {
+    wallets: walletModel.$wallets,
+    proxyAccount: $proxyAccount,
+  },
+  ({ wallets, proxyAccount }) => {
+    if (nullable(proxyAccount)) return null;
+
+    return walletUtils.getWalletById(wallets, proxyAccount.walletId);
+  },
+);
+
+const $proxyBalance = combine(
+  {
+    proxyAccount: $proxyAccount,
+    balances: balanceModel.$balances,
+    network: $networkStore,
+  },
+  ({ proxyAccount, balances, network }) => {
+    if (nullable(proxyAccount) || nullable(balances) || nullable(network)) return ZERO_BALANCE;
+
+    const balance = balanceUtils.getBalance(
+      balances,
+      proxyAccount.accountId,
+      network.chain.chainId,
+      network.asset.assetId.toString(),
+    );
+
+    return transferableAmount(balance);
+  },
+);
+
+sample({
+  clock: $route.updates,
+  source: $api,
+  filter: (api, route) => nonNullable(api) && route.some(accountUtils.isMultisigAccount),
+  fn: (api, route) => ({
+    api: api!,
+    threshold: route.find(accountUtils.isMultisigAccount)!.threshold,
+  }),
+  target: getMultisigDepositFx,
+});
+
+sample({
+  clock: getMultisigDepositFx.doneData,
+  target: $multisigDeposit,
+});
+
+const $canSubmit = combine(
+  {
+    isFormValid: form.$isValid,
+    isFeeLoading: $pendingFee,
+  },
+  ({ isFormValid, isFeeLoading }) => {
+    return isFormValid && !isFeeLoading;
+  },
+);
+
+// Fields connections
+sample({
+  clock: formInitiated,
+  target: form.reset,
+});
+
+sample({
+  clock: formInitiated,
+  filter: ({ chain, shards }) => nonNullable(getRelaychainAsset(chain.assets)) && shards.length === 1,
+  fn: ({ chain, shards }) => ({
+    initiator: shards.at(0) ?? null,
+    networkStore: { chain, asset: getRelaychainAsset(chain.assets)! },
+  }),
+  target: spread({
+    initiator: form.fields.initiator.change,
+    networkStore: $networkStore,
+  }),
+});
+
+// Submit
+sample({
+  clock: form.submit.doneData,
+  source: {
+    network: $networkStore,
+    route: $route,
+    fee: $fee.map((fee) => fee.toString()),
+    multisigDeposit: $multisigDeposit,
+    selectedSignatory: form.fields.signatory.$value,
+  },
+  filter: ({ network, selectedSignatory }) => {
+    return nonNullable(network) && nonNullable(selectedSignatory);
+  },
+  fn: ({ selectedSignatory, network, ...fee }, formData) => {
+    const { initiator } = formData;
+
+    return {
+      ...fee,
+      totalFee: fee.fee.toString(),
+      chain: network!.chain,
+      initiator: initiator!,
+      signatory: selectedSignatory!,
+    };
+  },
+  target: formSubmitted,
+});
+
+const $txWrappers = createTxWrappers({
```

Comment:
rebase branch to the latest feat/cas branch head and remove tx wrappers usage completely